conftest.py

```diff
@@ -136,6 +136,81 @@ def test_table(request, sql_server):  # noqa: C901
             hook.run(f"DROP SCHEMA IF EXISTS {table.metadata.schema} CASCADE;")
 
 
+@pytest.fixture
+def table_fixture(request, sql_server):  # noqa: C901
+    tables = []
```

Comment:
@utkarsharma2 since we've renamed this, would it be possible for the `table_fixture` to be closer to the `database_table_fixture`, using the `Database` class? I put quite some effort to make sure tear down was working properly for all databases!

Reply:
Sure, we can I'll make this change.

Reply:
@tatiana I'll raise a separate PR for this.